docs/resource/process.md

```diff
@@ -156,13 +156,19 @@ Examples for some JavaScript runtimes
 
 ### .NET Runtimes
 
-TODO(<https://github.com/open-telemetry/opentelemetry-dotnet/issues/1281>): Confirm the contents here
+.NET instrumentation should fill in the values by following values:
 
-| Value | Description |
-| --- | --- |
-| `dotnet-core` | .NET Core, .NET 5+ |
-| `dotnet-framework` | .NET Framework |
-| `mono` | Mono |
+- `process.runtime.name` - Fill in the value by the name of runtime.
+- `process.runtime.version` - Fill in the value of `System.Environment.Version`.
+- `process.runtime.description` - Fill in the values of `System.Runtime.InteropServices.RuntimeInformation.FrameworkDescription`.
+
+Examples for some .NET runtimes
+
+| Name | `process.runtime.name` | `process.runtime.version` | `process.runtime.description` |
+| --- | --- | --- | --- |
+| .NET Framework | .NET Framework | 4.0.30319.42000 | .NET Framework 4.8.9195.0 |
```

Comment:
In my experience the CLR version vs. Framework version can confuse people - especially when they look at observability data from mixed environment and aren't very familiar with .NET.

Would it make sense to call out this difference as a short note?

Reply:
Yes, first feedback receiver, expected more today.

For now, the recommendation is to use .NET Fx version instead of CLR version. PR contains these change.

Reply:
SIG recommendation changed for .NET Framework. It should be calculated based on the registry values:
6c13a65197245803f510c5dd7938eb0f3dc6fc22